Bootstraps/Symfony.php

```diff
@@ -49,16 +49,24 @@ public function getApplication()
         $appAutoLoader = './app/autoload.php';
         if (file_exists($appAutoLoader)) {
             require $appAutoLoader;
-        } else {
+        } elseif (file_exists('./vendor/autoload.php')) {
             require './vendor/autoload.php';
         }
+        if(class_exists('\AppKernel')) {
+            $app = new \AppKernel($this->appenv, $this->debug);
+        } else {
+            if(file_exists('./app/bootstrap.php.cache') && file_exists('./app/AppKernel.php')) {
```

Comment:
I'm not sure if this is even the correct way to check if `\AppKernel` exists, because when `class_exists` returns false, it means the autoloader isn't configured correctly, which also means all namespaces and classes inside `app/AppKernel.php` can't be resolved, so require this file will fail. So I think `'./app/AppKernel.php'` should be excluded here completely.